wrappers/vroom.rb

```diff
@@ -29,31 +29,42 @@ def initialize(hash = {})
 
     def solver_constraints
       super + [
+        # Costs
+        :assert_homogeneous_costs,
+        :assert_no_cost_fixed,
         :assert_vehicles_objective,
-        :assert_vehicles_only_one,
-        :assert_vehicles_start_or_end,
-        :assert_vehicles_no_end_time_or_late_multiplier,
-        :assert_services_no_capacities,
-        :assert_services_no_skills,
-        :assert_services_no_timewindows,
-        :assert_services_no_priority,
-        :assert_no_shipments,
-        :assert_no_direct_shipments,
-        :assert_matrices_only_one,
+
+        # Problem
         :assert_correctness_matrices_vehicles_and_points_definition,
-        :assert_one_vehicle_only_or_no_sticky_vehicle,
+        :assert_no_evaluation,
+        :assert_no_partitions,
         :assert_no_relations,
-        :assert_vehicles_no_duration_limit,
-        :assert_no_value_matrix,
+        :assert_no_subtours,
         :assert_points_same_definition,
+        :assert_single_dimension,
+
+        # Vehicle/route constraints
+        :assert_homogeneous_router_definitions,
+        :assert_matrices_only_one,
         :assert_no_distance_limitation,
-        :assert_no_subtours,
-        :assert_no_planning_heuristic,
-        :assert_no_evaluation,
+        :assert_vehicles_no_duration_limit,
+        :assert_vehicles_no_force_start,
+        :assert_vehicles_no_late_multiplier_or_single_vehicle,
+        :assert_vehicles_no_overload_multiplier,
+        :assert_vehicles_start_or_end,
+
+        # Mission constraints
+        :assert_no_direct_shipments,
+        :assert_no_exclusion_cost,
+        :assert_no_setup_duration,
+        :assert_services_no_late_multiplier,
```

Comment:
```suggestion
        :assert_missions_no_late_multiplier,
```
Should we check shipments too?